### DISCUSSION TITLE
Join two metrics with the same id label

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [open-telemetry/opentelemetry-collector-contrib#19366] [Q&A]
Is possible join/enrich metrics collected via prometheus reciever via id label? Similar to https://prometheus.io/docs/prometheus/latest/querying/operators/#group-modifiers

For example these metrics:
```
podman_container_info{id="19286a13dc23",image="docker.io/library/sonarqube:latest",name="sonar01",pod_id="",ports="0.0.0.0:9000->9000/tcp"} 1
```
and
```
podman_container_state{id="19286a13dc23"} 2
```
To have:
```
podman_container_state{id="19286a13dc23",image="docker.io/library/sonarqube:latest",name="sonar01",pod_id="",ports="0.0.0.0:9000->9000/tcp"} 2
```

Related:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18771

Thanks

**Comment by bimax** [comment]
I think you can use label_replace in your promql for this.
